core/src/main/java/org/apache/iceberg/BaseMetastoreTableOperations.java

```diff
@@ -309,65 +300,19 @@ protected enum CommitStatus {
    * @return Commit Status of Success, Failure or Unknown
    */
   protected CommitStatus checkCommitStatus(String newMetadataLocation, TableMetadata config) {
-    int maxAttempts =
```

Comment:
This needed to move to a common place so that other operations like View can also use it.